src/slc/mailrouter/tests/test_injection.py

```diff
@@ -3,7 +3,7 @@
 from Testing.makerequest import makerequest
 from zope.component.hooks import getSiteManager
 from zope.publisher.http import HTTPResponse
-from StringIO import StringIO
+from six import StringIO
```

Comment:
this should be `from io import StringIO`

Reply:
Why?

Reply:
The appropriate class to store text is io.StringIO, while the appropriate class to store bytes is io.BytesIO.
Down I see that the code does:
```
mail = safe_unicode(mail)
StringIO(mail)
```
So I think that everything in this method should be textish.

Reply:
AFAICT six.StringIO does exactly the same. The only point I see is that importing from io does not rely on a third party package.